routes/defRoute.js

```diff
@@ -3,12 +3,12 @@ const router = express.Router();
 
 const Defibrillator = require('../models/Defibrillator');
 
-router.get(`/`, async (req, res) => {
+router.get('/', async (req, res) => {
```

Comment:
It has been already changed in my last pull request, that Orest approved few minutes ago.